student-library/src/main/java/tds/student/services/abstractions/IContentService.java

```diff
@@ -50,4 +51,6 @@ IITSDocument getStimulusContent (long bankKey, long stimulusKey, AccLookup
   ITSMachineRubric parseMachineRubric (IITSDocument itsDocument, String language, RubricContentSource rubricContentSource) throws ReturnStatusException;
 
   void loadPageGroupDocuments (PageGroup pageGroup, AccLookup accLookup) throws ReturnStatusException;
+
+  Itemrelease getWordListItemRelease(String path) throws ReturnStatusException;
```

Comment:
Can we add JavaDocs?  I know they didn't do it originally but I figure we can add it for our new methods.